src/main/views/includes/support-links.njk

```diff
@@ -1,4 +1,18 @@
 <ul>
+  {% if featureToggles.testingSupport %}
```

Comment:
Why testing support is used to toggle on / off the support link? Testing support is always disabled in PROD. Please change condition!

Reply:
Could you elaborate on this please? @geek-soft

Reply:
Testing support feature toggle is used only to conditionally display features that support testing an application in non prod environments like changing response deadline or deleting drafts. Footer links has to be available in production environment when we are ready to have them (PB or sooner). Because of that we need to change condition and use different feature flag.